bucket/lightbulb.json

```diff
@@ -1,29 +1,29 @@
 {
-    "version": "1.6.4.1",
+    "version": "2.0",
     "description": "Continuously adjusts gamma, transitioning the display color temperature from cold blue to warm yellow.",
     "homepage": "https://github.com/Tyrrrz/LightBulb",
     "license": "GPL-3.0-or-later",
-    "url": "https://github.com/Tyrrrz/LightBulb/releases/download/1.6.4.1/LightBulb-Portable.zip",
-    "hash": "15ba9a6a9c890da8a787dc7af041349b66e179821ab7a28dc115f80c458853c4",
-    "pre_install": "if (!(Test-Path \"$persist_dir\\Configuration.dat\")) { Set-Content \"$dir\\Configuration.dat\" -Value '{}' -Encoding Ascii | Out-Null }",
+    "url": "https://github.com/Tyrrrz/LightBulb/releases/download/2.0/LightBulb.zip",
+    "hash": "72eb00bf6cda5dea68c4eb6fc5b0930f6f7a70d171ae9338309c1e32c5d54254",
+    "pre_install": "if (!(Test-Path \"$persist_dir\\Configuration.dat\")) { Set-Content \"$dir\\Configuration.dat\" -Value '{}' -Encoding Ascii }",
     "bin": [
         [
-            "LightBulb_Portable.bat",
-            "LightBulb"
+            "LightBulb.exe",
+            "LightBulb",
+            "-portable"
         ]
     ],
     "shortcuts": [
         [
-            "LightBulb_Portable.bat",
+            "LightBulb.exe",
             "LightBulb",
-            "",
-            "LightBulb.exe"
+            "-portable"            
         ]
     ],
     "persist": "Configuration.dat",
-    "notes": "If the gamma doesn't change under 6600K, run '$dir\\Manual_Gamma_Registry_Fix.reg'.",
```

Comment:
Why this was removed? It was fixed or??

Reply:
> Added a check at startup that will verify that extended gamma range is enabled on your computer and, if not, prompt you to enable it automatically. This also means that portable distributions of LightBulb no longer require you to change registry yourself.

See https://github.com/Tyrrrz/LightBulb/releases/tag/2.0